### PR TITLE
Add test cases for Ivan's commands

### PR DESCRIPTION
**Pull request by ivantjh**
Manual tests added for the below
- autocomplete
- `add-picture` 
- `del-picture`
- `add-meeting`
- `del-meetng`
- `clear`
- `help`

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -559,22 +568,55 @@ person on the list is born before 12-12-2020.
...
 
+Prerequisites: Download the sample picture [here](https://github.com/AY2021S2-CS2103T-W14-1/tp/raw/master/src/test/data/PictureTest/picture.jpg). 
+List all persons using the `list` command. There is at least a person present in the list.  
+
+1. Adding a picture to an existing person
+   
+    * These arguments should be replaced with the correct details.
+        * `PICTURE_FILE_PATH` - the file path of the sample picture, e.g. `Users/bob/Desktop/picture.jpg` for macOS
+        * `INVALID_FILE_PATH` - any invalid file path, e.g. `asdf123`
+        * `FILE_PATH_WITH_WRONG_FILE_EXTENSION` - a valid file path that points to an actual file but is not an image (Refer to UG `add-picture` command for a list of valid file extensions) <br><br>
```
Comment (lue97) [inline docs/DeveloperGuide.md]: maybe link add-picutre?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -559,22 +568,55 @@ person on the list is born before 12-12-2020.
...
+       Expected: Similar to previous
+         
 ### Clearing all entries: `clear`
 
+Prerequisites: List all persons using the `list` command. There is at least a person present in the list.
+
+1. Clearing all data
+    
+    1. Test case: `clear` <br>
+       Expected: All persons deleted from the app.
```
Comment (lue97) [inline docs/DeveloperGuide.md]: remove groups as well

**Review by lue97 — APPROVED** [review]
lgtm

**Review by Assyarul — APPROVED** [review]
lgtm.

**Pull request merged**